.golangci.toml

```diff
@@ -0,0 +1,37 @@
+[run]
```

Comment:
Shall we switch the Makefile to use golangci? Otherwise this file is currently unused.

Reply:
I removed that file.